simulator/scheduler/plugin/wrappedplugin.go

```diff
@@ -665,8 +689,8 @@ func (w *wrappedPlugin) PreBind(ctx context.Context, state *framework.CycleState
 
 func (w *wrappedPlugin) Bind(ctx context.Context, state *framework.CycleState, pod *v1.Pod, nodename string) *framework.Status {
 	if w.originalBindPlugin == nil {
-		// return nil not to affect scoring
-		return nil
+		// return skip not to affect scoring
```

Comment:
should be merged after https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/282

/hold

Reply:
/unhold